### PR TITLE
Added Student Features

### PR DESCRIPTION
**Pull request by Pankaj-Talesara**
Added animation to feature section; 
Fixed Padding and Margin

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,13 +58,13 @@ <h2>
         </h2>
 
         <div class="radio-group flex justify-center ">
-            <input type="radio" class="button radio" id="teacher" name="joinee" value="teachers" checked>
-            <label for="teacher">I'm a Teacher</label>
+            <input type="radio" class="button radio" id="teacher-btn" name="joinee" value="1" checked onclick="handleFeatureSwitch('teacher')">
```
Comment (kuldeepGkmit) [inline index.html:61]: find a better way to do it

File: index.html [hunk 2]
```diff
@@ -58,13 +58,13 @@ <h2>
...
+            <label for="teacher-btn">I'm a Teacher</label>
 
-            <input type="radio" class="button radio" id="student" name="joinee" value="students">
-            <label for="student"> I'm a Student </label>
+            <input type="radio" class="button radio" id="student-btn" name="joinee" value="2"  onclick="handleFeatureSwitch('student')">
```
Comment (kuldeepGkmit) [inline index.html:64]: same here

File: index.html [hunk 3]
```diff
@@ -115,6 +115,56 @@ <h3 class="feature-title flex-row">
                     </div>
                 </div>
             </div>
+
```
Comment (kuldeepGkmit) [inline index.html:118]: multiple div with no class or style should not be there

File: index.html (hunk 1 above)
Comment (Pankaj-Talesara) [reply index.html]: Have made the changes for both radio buttons. 
Commit: https://github.com/Pankaj-Talesara/QuizMatics/pull/7/commits/567f0281aced90722336f692a3756c476e2379f3

Comment (kuldeepGkmit) [reply index.html]: still need a better way

**Pull request merged**